src/common/ChatterSet.hpp

```diff
@@ -39,6 +39,12 @@ class ChatterSet
     /// are in mixed case if available.
     std::vector<QString> filterByPrefix(const QString &prefix) const;
 
+    /// Get all recent chatters, except the one with the matched username.
+    /// The first pair element contains the username in lowercase,
+    /// while the second pair element is the original case.
```

Comment:
This should mention that the username is the lowercase name (or: the login name).